src/components/Popups/PopupCursor.scss

```diff
@@ -0,0 +1,23 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.ba-PopupCursor {
+    .ba-Popup-arrow {
+        display: none;
+    }
+
+    .ba-Popup-content {
+        padding: 3px 6px;
+        color: $white;
+        font-size: 11px;
+        font-family: Lato, sans-serif;
```

Comment:
Should we use the `common-typography` mixin for `.ba-Popup` in its base stylesheet?